fastseq/optimizer/fairseq/generate.py

```diff
@@ -272,7 +272,7 @@ def add_generation_args_v1(parser):
     group = original_add_generation_args(parser)
     # fmt: off
     group.add_argument(
-        '--post-process-workers',
+        '--postprocess_workers',
```

Comment:
in fairseq arguments, "-" is used to connect words. It is a convention. Let's keep it.

Reply:
this argument was used in benchmark tests, please search and update accordingly.